src/test/groovy/org/prebid/server/functional/tests/BidderParamsSpec.groovy

```diff
@@ -809,7 +816,39 @@ class BidderParamsSpec extends BaseSpec {
             gpid == impExt.gpid
             skadn == impExt.skadn
             tid == impExt.tid
+            prebid.adUnitCode == impExt.prebid.adUnitCode
+        }
+    }
+
+    def "PBS shouldn't emit warning and proceed auction when all ext.prebid.bidderParams fields known for PBS"() {
```

Comment:
Please correct test name, since for now sounds like we shouldn't proceed auction